docs/ref/models/querysets.txt

```diff
@@ -2572,23 +2572,25 @@ in the database <database-time-zone-definitions>`.
 hour
 ~~~~
 
-For datetime fields, an exact hour match. Allows chaining additional field
```

Comment:
Do you think I should also add 1.7 version changed bits so that it's clear it was actually added in 1.7 for the non SQLite backends?